### PR TITLE
Add import_url to project

### LINKED ISSUE #42: [feature request] support importing projects in the `gitlab_project` resource
Hi,

I am looking at using the `gitlab` provider to manage a variety of projects, and wondering how reasonable it is to use this resource when the project needs to be imported. EG, in gitlab, when you create a new project, you can select to import that project from an HTTP or git/etc URL. When creating the project with the Terraform resource, I would like to be able to specify the URL the project should be initially imported from. This import is a one-time action, so IDK if it makes sense for the provider to support this, but it seems worth asking about.

Thanks!

### PR DESCRIPTION
**Pull request by joepvd**
This adds `import_url` to the options to create a project, and implements the suggested acceptance test from #142.

Supersedes #142
Resolves #42

### CODE REVIEW

**Comment by joepvd** [comment]
Friendly ping :)

File: gitlab/resource_gitlab_project.go [hunk 1]
```diff
@@ -290,13 +298,20 @@ func resourceGitlabProjectCreate(d *schema.ResourceData, meta interface{}) error
...
+	var project *gitlab.Project
 	client := meta.(*gitlab.Client)
 	log.Printf("[DEBUG] read gitlab project %s", d.Id())
 
 	project, _, err := client.Projects.GetProject(d.Id(), nil)
 	if err != nil {
 		return err
 	}
+	for project.ImportStatus == "started" {
+		project, _, err = client.Projects.GetProject(d.Id(), nil)
```
Comment (roidelapluie) [inline gitlab/resource_gitlab_project.go]: maybe add a sleep? or use terraform builtin functions `stateConf := &resource.StateChangeConf` ?

**Comment by roidelapluie** [comment]
Added 1 comment.

**Comment by durnerj** [comment]
Would love to have this Feature right now, any Updates or an ETA on this?

**Comment by jgreat** [comment]
Any updates on this feature?

**Comment by ringods** [comment]
Hello @joepvd, I became a maintainer of this provider not so long ago. I'm going over the pull requests backlog one by one.

If you are still interested in getting this contribution merged, may I ask you the following?
* review the comment @roidelapluie made to your code on 11 September 2019
* do a proper `git rebase` as your work currently has merge conflicts with what is on `master`.

Thanks.

**Comment by pandom** [comment]
Hey,

Please let me know what blockers exist here. If it is a rebase, can we rebase please? Otherwise can we resubmit this PR? Thanks, Burkey

**Pull request closed without merging**